### DISCUSSION TITLE
The taskbar is sometimes covered by maximized windows

### DISCUSSION THREAD

**Discussion by martinezgjuan** [valinet/ExplorerPatcher#1268] [Q&A]
It has happened a few times. Right now, for example, Chrome is covering the task bar.
Restarting the PC fixes the issue. Looks like a bug.
I'm using version 22621.1413.54.5

**Comment by ldx00** [comment]
I have exactly the same problem. Can confirm that this happens after my screens go to sleep and after some amount of idle time. It happens without fail to me after the PC has been idle for around 30 minutes (well after the screens have gone to sleep, so this itself is not the only trigger). I did have another taskbar modification tool installed though and maybe this also causes a problem (WinAerotweaker), not sure. I can also confirm it does NOT happen if I switch to Windows 11 taskbar, only when I select Windows 10 style taskbar. I'm on 22H2 22621.1265 on an AMD 6800H laptop.

Temporary workaround is either to restart explorer or my preferred method
press Ctrl + Shift + Win + B which refreshes the displays or something

This problem's been around for a while and I've had it since I got explorer patcher at the beginning of the year. Happy to do any diagnostics as this is really annoying (but is the only thing currently wrong), otherwise a great program and the only one that wouldn't crash explorer since the update to 22H2. Keep up the good work!

**Comment by pyrates999** [comment]
You could start by uninstalling WinAerotweaker and retesting this scenario to see if it happens again.

**Comment by ldx00** [comment]
I had a bit more of a read since yesterday and I did a restore default settings of Explorer Patcher and the issue seems to be resolved! It does sometimes take a little while to show though, up so I'll keep an eye on it but so far it looks good.
